src/yb/tserver/tablet_server_main.cc

```diff
@@ -141,6 +143,13 @@ int TabletServerMain(int argc, char** argv) {
   FLAGS_webserver_port = TabletServer::kDefaultWebPort;
   FLAGS_redis_proxy_webserver_port = RedisServer::kDefaultWebPort;
   FLAGS_cql_proxy_webserver_port = CQLServer::kDefaultWebPort;
+
+  string host_name;
+  if (GetHostname(&host_name).ok()) {
+    FLAGS_metric_node_name = (host_name) + ":" + std::to_string(TabletServer::kDefaultWebPort);
```

Comment:
Minor: You can use `strings::Substitute()` like above.

Reply:
Will do, thanks.